corehq/util/hmac_request.py

```diff
@@ -21,7 +21,7 @@ def get_hmac_digest(shared_key, data):
     return digest.decode('utf-8')
 
 
-def validate_request_hmac(setting_name, ignore_if_debug=False):
+def validate_request_hmac(setting_name, ignore_if_debug=False, audit_user='system'):
```

Comment:
Are you planning to pass the new param in the future?

Reply:
I wasn't sure, but I guess not. I put it as a param because it seems (abstractly) like something that the caller would want control over, but I don't have future plans to pass in other values here